package.json

```diff
@@ -7,11 +7,11 @@
     "addons-linter": "bin/addons-linter"
   },
   "scripts": {
-    "start": "node -e \"require('grunt').cli()\" null start",
```

Comment:
It does (and passes Travis-CI). See http://gruntjs.com/blog/2016-02-11-grunt-1.0.0-rc1-released#npm-scripts

It took me a while to read the npm scripts and figure out what was going on with Grunt `cli()` and `null`s. Alternatively, we could have just done an `npm install grunt-cli -g` in the .travis.yml, but the above change in grunt@1 works really well.